### PR TITLE
Feature/frontend

### PR DESCRIPTION
**Pull request by fpdotmonkey**
I've made a static webpage that shows the status of each of the 4 patients.  It doesn't yet dynamically display patient state; that'll be a future pull request.

To verify that it works, open `02_Development/Display/Static/index.html` in a browser, and there should be a big table that shows off a sample patient state.  There will also be a bit of text below that.  That'll be removed in a future pull request.

### CODE REVIEW

**Comment by thehannahfink** [comment]
It’s looking great so far! Awesome job @fpdotmonkey!

A few notes:

1. I think there’s something funky going on with the border around the turquoise arrows.

2. Can we align the units in the left (turquoise) column with the bottom of the other text?

3. Can we decrease the stroke thickness around the readings? The stroke around the big “1, 2, 3, 4” is perfect, but needs to be a little thinner around the smaller text.

4. Is it possible to have the top row rectangles bleed all the way to the edge? If not, let’s shorten those rectangles and leave an 18px white border around the edge.

**Comment by fpdotmonkey** [comment]
Here's the first version that got pushed https://files.slack.com/files-pri/TUTSYURT3-F017MA9UE30/image.png

**Comment by fpdotmonkey** [comment]
Ok new version
image

**Comment by thehannahfink** [comment]
Looking better. Still seems like the borders around the turquoise arrows are uneven. Since there's still a sliver of white above the top row, let's just knock those rectangles down and leave an 18px border.

**Comment by fpdotmonkey** [comment]
image

**Comment by mmroden** [comment]
So this is not a testable commit?  If it's not testable, then I think that Hannah has to be the final arbiter

**Comment by fpdotmonkey** [comment]
That's what we're doing

**Comment by thehannahfink** [comment]
Is there any way to fix that border in the first row? 
Screen Shot 2020-07-28 at 2 24 00 PM

**Comment by fpdotmonkey** [comment]
image

**Comment by thehannahfink** [comment]
It is very very close but not quite. It looks like you've put a slightly larger/darker arrow graphic behind the lighter arrow to achieve the border. Is that what you did with the rectangles? I wonder if it would make more sense to just make the arrows rectangles so the border can be as clean as it is in the rest of the display graphics.

**Comment by fpdotmonkey** [comment]
Because the row headers are  funny shape, they get treated different from the rectangles.  So yes I'm putting a different larger shape behind the row headers, but I'm using the CSS `border` property for the rectangles.  So it would definitely be easier to make the row headers rectangles.  Let me try again with the row header border, and it you're still not satisfied, I'll make it a rectangle.

**Comment by fpdotmonkey** [comment]
image

**Comment by fpdotmonkey** [comment]
Hannah Fink  15:09
The border looks great. Looks like the sizing is a little off though. Is this no longer 800x480?

Fletcher  15:13
It ends up being that the sizes you called out don’t alway come out to the same size in HTML. Im guessing you want them wider? (edited) 

Hannah Fink  15:16
Ah yeah there seems to be some spatial issues with the text
15:16
most noticeably here

Fletcher  15:17
That said, I think the data cells are as narrow as they could be. To make the row headers wider id have to eat into the whitespace

Hannah Fink  15:17
screen_shot_2020-07-28_at_3 16 56_pm


15:17
can we make the text a tiny bit smaller instead?

Fletcher  15:17
Sure

**Comment by fpdotmonkey** [comment]
image

**Comment by thehannahfink** [comment]
That's looking great. Can you knock down the font size of the readings just a tiny bit? Other than that I think we're good to go.
Screen Shot 2020-07-28 at 3 40 21 PM

**Comment by fpdotmonkey** [comment]
image

**Comment by thehannahfink** [comment]
Perfect! It's good to go
